src/components/annotate/sidePanel/SidePanel.tsx

```diff
@@ -110,14 +110,45 @@ const SidePanel = (props: Props) => {
         if (annotationFormToggle || !isEmpty(editAnnotation)) {
             dispatch(setAnnotationFormToggle(false));
             dispatch(setEditAnnotation({} as Annotation));
-        } else if (annotateTarget.targetProperty.name) {
-            ShowWithAllAnnotations();
+
+            if (annotateTarget.targetProperty.name && annotateTarget.targetProperty.type !== 'superClass') {
```

Comment:
Is it really needed to call this a `superClass`? Or can the targetType just be empty with empty indicating that it is about the whole object?

Reply:
Well, I tried that at first, but I think I could not get the logic to work with each other, as there are loads of conditionals in the side panel, checking what to display. Also, this type is local variable, so its not ingested into the annotation object itself or something like that, I get its not the prettiest solution right now, but I rather fix another time, as I think the annotation form needs to be split up into more components to make it better.

Reply:
Ok maybe create a task for it too look again some time in the future

Reply:
https://naturalis.atlassian.net/browse/DD-257?atlOrigin=eyJpIjoiZjRlNTA1ZTFmOTM2NGUwNDk4YmIyOGY3MGVmYTVmMTUiLCJwIjoiaiJ9